opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -1167,16 +1169,37 @@ def get_tracer(
             logger.error("get_tracer called with missing module name.")
         if instrumenting_library_version is None:
             instrumenting_library_version = ""
+
+        kwargs = {
+            "message": (
+                "Call to deprecated method __init__. \(You should use "
+                "InstrumentationScope\) -- Deprecated since version 1.11.1."
+            ),
+            "category": DeprecationWarning,
+            "module": "opentelemetry.sdk.trace",
+        }
+
+        # FIXME: Remove when 3.7 is no longer supported.
+        # For some reason the lineno argument in 3.7 does not seem to work.
+        # This is still safe enough because we also filter by message and
+        # module.
+        if version_info.minor > 7:
+            kwargs["lineno"]: 1191
+
+        filterwarnings("ignore", **kwargs)
+
+        instrumentation_info = InstrumentationInfo(
+            instrumenting_module_name,
+            instrumenting_library_version,
+            schema_url,
+        )
+
         return Tracer(
             self.sampler,
             self.resource,
             self._active_span_processor,
             self.id_generator,
-            InstrumentationInfo(
-                instrumenting_module_name,
-                instrumenting_library_version,
-                schema_url,
-            ),
+            instrumentation_info,
```

Comment:
Nice!
Can you do the same for InstrumentationScope?

Reply:
Curious as to why the same needs to be applied for `InstrumentationScope`?

Reply:
It's not needed, And it's not really related to this PR
It's more like code style, just a suggestion.

Reply:
If what you mean is to instantiante `InstrumentationScope` in a separate variable, then passing that variable to the constructor, it would mean creating a single-use variable which I avoid as much as possible since it is unnecessary extra code.